packages/opentelemetry-metrics/src/types.ts

```diff
@@ -15,45 +15,15 @@
  */
 
 import { LogLevel } from '@opentelemetry/core';
-import { Logger, ValueType } from '@opentelemetry/api';
+import * as api from '@opentelemetry/api';
 import { MetricExporter } from './export/types';
 import { Resource } from '@opentelemetry/resources';
 import { Batcher } from './export/Batcher';
 
-/** Options needed for SDK metric creation. */
-export interface MetricOptions {
```

Comment:
Where did the metric options move to?

Reply:
We have the same metrics options interface in api and the duplicated in metrics. Yet we were referencing in metrics those from api, which causes more confusions. Why do we need too have both then if the api is enough as you can see.

Reply:
This is intentional in order to get ride of Non-Null Assertion Operator! and/or assigning default value in the pipeline.

Related comment: #415 (comment)